Cargo.toml

```
@@ -17,7 +17,7 @@ reqwest = { version = "0.12.5", default-features = false, features = [
     "rustls-tls",
     "brotli",
     "gzip",
-    "http2"
+    "http2",
```

```suggestion
    "http2",
    "socks",
```

Well, one last and minor suggestion I think you forgot to add the socks feature here. So I would suggest adding it. Other than everything looks good. :+1: :slightly_smiling_face: .